### ISSUE TITLE
Docs should emphasize loss functions with logits

### ISSUE THREAD

**Issue by dsevero** [torchgan/torchgan#116]
I noticed that [minimax_discriminator_loss](https://github.com/torchgan/torchgan/blob/master/torchgan/losses/functional.py#L42) uses `binary_cross_entropy_with_logits` (cf. `binary_cross_entropy`) which "combines a Sigmoid layer and the BCELoss" [according to the docs](https://pytorch.org/docs/stable/nn.html#bcewithlogitsloss).

This is a good practice when it comes to numerical stability, but maybe emphasizing this in the docs would be a good idea to prevent users from inserting a sigmoid as the final non-linearity in custom models. Maybe even raising a warning if we can somehow detect this.
